integration-tests/chopsticks/src/network/spiritnet.ts

```diff
@@ -28,11 +37,7 @@ export function setGovernance(addr: string[]) {
 
 /// Spiritnet ParaId
 export const paraId = 2086
+export const KILT = { Concrete: { parents: 0, interior: 'Here' } }
```

Comment:
The naming of this could be improved. What this is, is the `Here` location within the context of the chain itself. So perhaps just call it `HERE`?